src/server/database/index.js

```diff
@@ -3,7 +3,9 @@ require('dotenv').config()
 
 let MONGO_URI
 if (process.env.LOCAL_DB) {
-  MONGO_URI = 'mongodb://localhost:27017/dev'
+  MONGO_URI = 'mongodb+srv://m001-student:m001-mongodb-basics@sandbox.wepyv.mongodb.net/?retryWrites=true&w=majority'
```

Comment:
I don't think this should be changed?

Reply:
This is OK to change locally (it just points the db to a personal Atlas instance instead of a locally running mongodb database), but it shouldn't be committed.